publisher-sdk/src/main/java/com/criteo/publisher/logging/AppBiddingLogMessage.kt

```diff
@@ -0,0 +1,38 @@
+/*
+ *    Copyright 2020 Criteo
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+package com.criteo.publisher.logging
+
+import android.util.Log
+import com.criteo.publisher.integration.Integration
+
+internal object AppBiddingLogMessage {
+
+  @JvmStatic
+  fun enrichAdObjectWithNoBid(integration: Integration) = LogMessage(message = {
```

Comment:
I would rename as: `createEnrichAdObjectWithNoBidLog`. (otherwise, it's really confusing to me as it seems the method will do that action).